Makefile

```diff
@@ -65,29 +65,13 @@ develop: test.requirements
 piptools: ## install pinned version of pip-compile and pip-sync
 	pip install -r requirements/pip-tools.txt
 
-COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
-.PHONY: $(COMMON_CONSTRAINTS_TXT)
-$(COMMON_CONSTRAINTS_TXT):
-	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
```

Comment:
One reason I prefer to just directly reference this file rather than download and update it when running `make upgrade` is that the base docker images for notes don't have wget. So we end up hitting the `|| touch` path, and local engineers trying to run make upgrade may have out of date constraints.